Src/main.c

```diff
@@ -131,51 +122,39 @@ int main(void)
   /* Init scheduler */
   osKernelInitialize();
 
-  /* USER CODE BEGIN RTOS_MUTEX */
-  printf("Main func\n");
-
-  /* USER CODE END RTOS_MUTEX */
-
-  /* USER CODE BEGIN RTOS_SEMAPHORES */
-
-  /* USER CODE END RTOS_SEMAPHORES */
-
-  /* USER CODE BEGIN RTOS_TIMERS */
-
-  /* USER CODE END RTOS_TIMERS */
-
-  /* USER CODE BEGIN RTOS_QUEUES */
-
-  /* USER CODE END RTOS_QUEUES */
-
-  /* Create the thread(s) */
-  /* creation of THREAD1 */
-  //THREAD1Handle = osThreadNew(LED_Thread1, NULL, &THREAD1_attributes);
-
-
-    status = xTaskCreate(LED_Thread1, "Task1", 500, &s1, 2, &task1_handle);
-    status1 = xTaskCreate(LED_Thread2, "Task2", 500, "Task-2 is running", 2, &task2_handle);
-    configASSERT(status == pdPASS);
-    configASSERT(status1 == pdPASS);
-
-    printf("Main Num :%d\n",s1.num);
 
-
-  /* creation of THREAD2 */
-  //THREAD2Handle = osThreadNew(LED_Thread2, NULL, &THREAD2_attributes);
-
-  /* USER CODE BEGIN RTOS_THREADS */
-  /* USER CODE END RTOS_THREADS */
-
-  /* USER CODE BEGIN RTOS_EVENTS */
-  /* add events, ... */
-  /* USER CODE END RTOS_EVENTS */
+  /* The queue is created to hold a maximum of 5 values, each of which is
+   large enough to hold a variable of type Data_t ). */
+   //xQueue = xQueueCreate( 3, sizeof( Data_t )) ;
+   xPointerQueue = xQueueCreate( 5, sizeof( char * ) );
+
+   if( xPointerQueue != NULL )
+   {
+   /* Create two instances of the task that will send to the queue. The
```

Comment:
Still code indentation in not proper.
You can enable trailing whitespace visibility in IDE from editors configuration.

Reply:
enable the whitespace option

Reply:
Made the changes